materialize-sqlserver/docker-compose.yaml

```diff
@@ -2,19 +2,19 @@ version: "3.7"
 
 services:
   sqlserver:
-    image: 'mcr.microsoft.com/mssql/server:2017-latest'
+    image: 'mcr.microsoft.com/mssql/server:2022-latest'
```

Comment:
Couldn't get the 2017 version to work on the newer ubuntu runner. The snapshot change is because this version uses `varchar` instead of `nvarchar` by default.